docs/old-reference-guide/modules/messaging-concepts/pages/supported-parameters-annotated-handlers.adoc

```diff
@@ -19,17 +19,17 @@ reading xref:message-handler-customization-guide::index.adoc[this] section.
 By default, `@CommandHandler` annotated methods allow the following parameter types.
 
 |===
-|Parameter designation |Purpose 
+|Parameter designation |Purpose
 
 |The first parameter |The first parameter is always the payload of the command message. It may also be of type: `Message` or `CommandMessage`, if the `@CommandHandler` annotation explicitly defined the name of the command the handler can process. By default, a command name is the fully qualified class name of the command its payload.
-|type: `MetaData` |Will contain the entire metadata of a `CommandMessage`. 
+|type: `MetaData` |Will contain the entire metadata of a `CommandMessage`.
 |annotated with `@MetaDataValue` |Will resolve the metadata value with the key as indicated on the annotation. If `required` is `false` (default), `null` is passed when the metadata value is not present. If `required` is `true`, the resolver will not match and prevent the method from being invoked when the metadata value is not present.
 |type: `Message`, or type: `CommandMessage` |Will get the complete message, with both the payload and the metadata. Resolving the entire `Message` is helpful if a method needs several metadata fields or other properties of the message.
 |type: `UnitOfWork` |Will get the current xref:unit-of-work.adoc[unit of work] injected. The `UnitOfWork` allows command handlers to register actions to be performed at specific stages of the unit of work or gain access to the resources registered with it.
 |type: `String` annotated with `@MessageIdentifier` |Will resolve the identifier of the handled `CommandMessage`.
-|type: `InterceptorChain` |Will resolve the chain of `MessageHandlerInterceptor`s for a `CommandMessage`. You should use this feature in conjunction with a `@CommandHandlerInterceptor` annotated method. For more specifics on this it is recommended to read xref:message-intercepting.adoc#command-handler-interceptor-annotation[this] section.
+|type: `MessageHandlerInterceptorChain` |Will resolve the chain of `MessageHandlerInterceptor`s for a `CommandMessage`. You should use this feature in conjunction with a `@CommandHandlerInterceptor` annotated method. For more specifics on this it is recommended to read xref:message-intercepting.adoc#command-handler-interceptor-annotation[this] section.
```

Comment:
I believe we should not change it in the old-reference-guide, because it's about AF4. Am I right @smcvb?

Reply:
Absolutely right! I'll adjust this in Simon's absence right away.